### ISSUE TITLE
Receiving error "ParseError: Unexpected token"

### ISSUE THREAD

**Issue by intelcoder** [msalsbery/OpenSeadragonViewerInputHook#10]
I am getting error on requiring OpenSeadragonViewerInputHook.

**Error message**
```
major: <%= viewerinputhookVersion.major %>,
               ^
ParseError: Unexpected token
```

**How I added to package json**
```
"OpenSeadragonViewerInputHook": "github:msalsbery/OpenSeadragonViewerInputHook",
```

**How I required js file**
```
require('OpenSeadragonViewerInputHook/src/viewerinputhook');
```

**Comment by mrmean007** [comment]
I got the same error

**Comment by msalsbery** [comment]
This should be fixed in 2.0.0, just released/posted
